src/sentry/incidents/subscription_processor.py

```diff
@@ -477,10 +467,12 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
                     ) and not self.check_trigger_matches_status(trigger, TriggerStatus.ACTIVE):
                         # If the value has breached our threshold (above/below)
                         # And the trigger is not yet active
-                        metrics.incr(
-                            "incidents.alert_rules.threshold.alert",
-                            tags=alert_triggered_tags,
-                        )
+                        metrics.incr("incidents.alert_rules.threshold.alert")
+                        if features.has(
+                            "organizations:workflow-engine-metric-alert-dual-processing-logs",
+                            self.subscription.project.organization,
+                        ):
+                            metrics.incr("dual_processing.alert_rules.fire")
```

Comment:
I don't need the detection type to filter out anomaly detection alerts (which are not yet migrated) because those get fired and resolved in the if block above this